### ISSUE TITLE
Test: tree and list aria-posinset and aria-setsize

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#29172]
- [x] win - @ramya-rao-a 
- [x] mac - @roblourens

Complexity: 2

Refs: https://github.com/Microsoft/vscode/issues/27900

We have added the `aria-posinset` and `aria-setsize` attributes to the list and quick open. By using developer tools inspect those HTML elements and verify these attributes are really there and they have appropriate values. You should also use a screen reader however I am not 100% sure how this should be reflected on a screen reader.

Examples where we use the list in vscode: extensions list, suggest widget.
To inspect quickOpen use this workbench setting `"workbench.quickOpen.closeOnFocusLost": false`

@joaomoreno feel free to edit

**Comment by roblourens** [comment]
I can't figure out how to get nvda to read "x of y" labels, but the aria attributes appear to be set correctly.

**Comment by ramya-rao-a** [comment]
@isidorn Are we to test both list and tree controls?

The title of this test item refers to both, description refers to only list control.

List controls in extension list has the required aria labels
Tree control in quick open has the required aria labels
But tree control in search results doesnt have the required labels.
I am guessing this is by design, but wanted to confirm nevertheless

**Comment by isidorn** [comment]
@ramya-rao-a yes, this is by design, sorry for not being more clear. For the tree we have implemented this only in quick open and we should check it there
@roblourens yes I am also not sure how to make nvda read that out, I think it is some special nvda command

Thanks for testing
